platform/darwin/src/MGLGeometry_Private.h

```diff
@@ -51,6 +51,7 @@ NS_INLINE mbgl::EdgeInsets MGLEdgeInsetsFromNSEdgeInsets(NSEdgeInsets insets) {
     @param latitude The latitude of the point at the center of the viewport.
     @param size The size of the viewport.
     @return An altitude measured in meters. */
+MGL_EXTERN
```

Comment:
Theoretically we shouldn't, since it should be all internal to the binary we're building, however the test target also includes this private header, and linking the lib to the test fails without exporting this symbol. Another way would be to not export this, and change the test so that it doesn't require this symbol, or to move this symbol to the public header.